src/main/blackbox/blackbox.c

```diff
@@ -1359,9 +1359,10 @@ static bool blackboxWriteSysinfo(void)
 
     const controlRateConfig_t *currentControlRateProfile = controlRateProfiles(systemConfig()->activeRateProfile);
     switch (xmitState.headerIndex) {
-        BLACKBOX_PRINT_HEADER_LINE("Firmware type", "%s",                   "Cleanflight");
+        BLACKBOX_PRINT_HEADER_LINE("Firmware type", "%s",                   "Betaflight");
         BLACKBOX_PRINT_HEADER_LINE("Firmware revision", "%s %s (%s) %s",    FC_FIRMWARE_NAME, FC_VERSION_STRING, shortGitRevision, targetName);
         BLACKBOX_PRINT_HEADER_LINE("Firmware date", "%s %s",                buildDate, buildTime);
+        BLACKBOX_PRINT_HEADER_LINE("DeviceUID", "%08x%08x%08x",             U_ID_0, U_ID_1, U_ID_2);
```

Comment:
Should we not make this human readable similar to log outputs?

Reply:
This represents `mcu_id` ?

Reply:
@blckmn this represents `mcu_id` parameter in CLI.

https://github.com/betaflight/betaflight/blob/fcecabf01e611f97b59c4af8138da14b1e7c8e4e/src/main/cli/cli.c#L3245

If this should be human readable it should be changed here as well, but imho it is used as a serial number.

Reply:
Ok - thats fine. All good :)